spec/models/vm/retirement_management_spec.rb

```diff
@@ -10,10 +10,10 @@
   end
 
   it "#retirement_check" do
+    User.current_user = user
```

Comment:
@d-m-u 
can we user User.with_user here similar to
https://github.com/ManageIQ/manageiq/blob/a9ea2507e0ff7b2939441f0a3be7bfbf4996f362/spec/lib/rbac/filterer_spec.rb#L263